r124/outref/ydb312_gtm8182d.txt

```diff
@@ -11,8 +11,17 @@ Starting Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_P
 Shutting down Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_PATH_/instance2
 Shutting down Primary Source Server Server in ##TEST_PATH##
 
+# Run ydb312gtm8182d.m to:
+#		- attempt to read from DB
```

Comment:
Sorry if I was not clear before. Lines 15017 are duplicates of lines 18, 20 and 23. I prefer the latter as they are displayed from within the M program. Please remove lines 18, 20 and 23.